sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,507 @@
+            ''--TBD--'' as "Youth in Care beyond age 19 Flag",
+            applications.data ->> ''outOfHighSchoolFor4Years'' as "Outside of highschool >4 years flag",
+            applications.data ->> ''whenDidYouGraduateOrLeaveHighSchool'' as "High School Graduation Date",
+            applications.data ->> ''fulltimelabourForce'' as "Recent Labour Force Participation Flag",
+            applications.data ->> ''addTrustContactToContactSABC'' as "Trusted Contact Identified (Yes/No)",
+            applications.data ->> ''relationshipStatus'' as "Marital Status",
+            --Application Financial Info
+            applications.data ->> ''taxReturnIncome'' as "Total income",
+            applications.data ->> ''partTimeAwardTypesToBeConsidered'' as "Grant-only flag",
+            applications.data ->> ''scholarshipsreceviedCosts'' as "Merit-based scholarship amount",
+            applications.data ->> ''parentvoluntarycontributionsCosts'' as "Voluntary parental contribution amount",
+            applications.data ->> ''governmentFundingCosts'' as "Government funding amount",
+            applications.data ->> ''nongovernmentfundingCosts'' as "Non-government funding amount",
+            applications.data ->> ''bcIncomeassistancefordisabilitiesCosts'' as "B.C. Income Assistance Amount",
+            applications.data ->> ''returntriphomeCost'' as "Relocation costs",
+            -- Application Financial Appeals Info
+            applications.data ->> ''exceptionalExpensesApplicationException'' as "Exceptional expense amount",
+            applications.data ->> ''daycareCosts11YearsOrUnder'' as "Daycare cost amount",
+            applications.data ->> ''totalUnsubsidizedDayCareCostsDuringClassHoursForTheProposedStudyPeriodForDisabledInfirmedDependents12YearsOfAgeAndOlderPanel'' as "Unsubsidized daycare amount",
+            ''--TBD--'' as "Child/Spousal Support amount",
+            ''--TBD--'' as "Living in parent house and paying living expenses amount",
+            applications.data ->> ''transportationcostsEstimate'' as "Additional transportation amount (as applicable)",
+            -- Parent Info Self-Reported by Student
+            applications.data ->> ''pleaseProvideAnEstimationOfYourParentsIncome'' as "Parental estimated income self-reported",
+            applications.data ->> ''studentParentNetAssests'' as "Parent value of canadian and foreign assets self reported",
+            applications.data ->> ''studentParentNetContribution'' as "Parental contribution self-reported",
+            -- Parent1 Info
+            parent1_users.first_name as "Parent 1 first name",
+            parent1_users.last_name as "Parent 1 last name",
+            parent1.contact_info -> ''address'' ->> ''postalCode'' as "Parent 1''s postal code",
+            parent1.supporting_data ->> ''totalIncome'' as "Parental income (line 15000)",
+            ''--TBD--'' as "Parental net annual income (calculated value)",
```

Comment:
Not a part of your ticket, but we have this 
 field `calculatedDataTotalParentNetIncome` in workflow which could be potential candidate to be noted down for upcoming ticket. 

![image](https://github.com/bcgov/SIMS/assets/54600590/2b068714-1e42-4a77-9643-239e5cf48a2e)